modules/gatk4/calculatecontamination/main.nf

```diff
@@ -9,7 +9,7 @@ process GATK4_CALCULATECONTAMINATION {
 
     input:
     tuple val(meta), path(pileup), path(matched)
-    val segmentout
+    val   segmentout
```

Comment:
@FriederikeHanssen I believe that this should go in modules.config in for $args

Reply:
yes I agree

Reply:
finally keeping it here as it generates extra output.

Reply:
ok, can you add a comment there for future me who will forget this

Reply:
no worries, I'm planning to add comments to explain more stuff around